src/openrct2-ui/windows/Map.cpp

```diff
@@ -561,12 +561,12 @@ static void window_map_scrollgetsize(rct_window* w, int32_t scrollIndex, int32_t
  *
  *  rct2: 0x0068D726
  */
-static void window_map_scrollmousedown(rct_window* w, int32_t scrollIndex, int32_t x, int32_t y)
+static void window_map_scrollmousedown(rct_window* w, int32_t scrollIndex, ScreenCoordsXY screenCoords)
 {
-    CoordsXY c = map_window_screen_to_map(x, y);
+    CoordsXY c = map_window_screen_to_map(screenCoords);
     int32_t mapX = std::clamp(c.x, 0, MAXIMUM_MAP_SIZE_TECHNICAL * 32 - 1);
     int32_t mapY = std::clamp(c.y, 0, MAXIMUM_MAP_SIZE_TECHNICAL * 32 - 1);
-    int32_t mapZ = tile_element_height({ x, y });
+    int32_t mapZ = tile_element_height({ screenCoords.x, screenCoords.y });
```

Comment:
This is mixing up `ScreenCoordsXY` and `CoordsXY` which you might want to chime in @duncanspumpkin

Reply:
Yes looks like a mistake should be mapx and mapy

Reply:
Done!